Build/phpstan-baseline.neon

```diff
@@ -66,74 +66,74 @@ parameters:
 			path: ../Classes/TcaDataGenerator/FieldGenerator/TypeFlex.php
 
 		-
-			message: "#^Method TYPO3\\\\CMS\\\\Styleguide\\\\TcaDataGenerator\\\\FieldGenerator\\\\TypeGroupDbAllowedBeUsers\\:\\:generate\\(\\) has parameter \\$data with no value type specified in iterable type array\\.$#"
+			message: "#^Method TYPO3\\\\CMS\\\\Styleguide\\\\TcaDataGenerator\\\\FieldGenerator\\\\TypeFolder\\:\\:generate\\(\\) has parameter \\$data with no value type specified in iterable type array\\.$#"
```

Comment:
phpstan baseline changes are a trade because of the renamings. So no real additions/removals. So this is a simple update to align the ignore patterns. Not related to this patch. However, we should address this in a dedicated patch to fix these issues. But not related to this patch. Fine with that.

Reply:
ack